deis.go

```diff
@@ -89,7 +89,7 @@ type Client struct {
 // controller is unsafe.
 //
 // If the SDK detects an API version mismatch, it will return ErrAPIMismatch.
-const APIVersion = "2.1"
+const APIVersion = "2.2"
```

Comment:
Nah this is the API version bump rather than the platform version bump. These are versioned separately, but because we've had new features in the API these past two releases they are identical. :)

Note that https://github.com/deis/controller/pull/737 also bumps the API version, and https://github.com/deis/workflow/pull/367 adds a v2.2 API reference doc.